python/tests/aggregate_tests/test_array_tbl.py

```diff
@@ -0,0 +1,17 @@
+from tests.aggregate_tests.aggtst_base import TstTable
+
+
+class aggtst_array_tbl(TstTable):
```

Comment:
this is fine, but array_agg actually does not need elements to be arrays themselves, it works for integers too.
I would suggest making column c2 just INT, and using that too.
If you want to get fancy you can also use a column c3 which is a map.

Reply:
I used only arrays since these are array type tests. I can add a MAP or INT type column if needed, though.

Reply:
yes, but the *elements* of an array can have any type